concept/type/plays.feature

```diff
@@ -0,0 +1,2123 @@
+    Then <root-type>(<type-name>) get plays(parentship:parent) get annotations contain: @<annotation>
+    Then <root-type>(<type-name>) set plays: contract:contractor
+    Then <root-type>(<type-name>) get plays(contract:contractor) get annotations is empty
+    When transaction commits
+    When connection open schema transaction for database: typedb
+    Then <root-type>(<type-name>) get plays(parentship:parent) get annotations contain: @<annotation>
+    When <root-type>(<type-name>) set plays: parentship:parent
+    Then <root-type>(<type-name>) get plays(parentship:parent) get annotations is empty
+    Examples:
+      | root-type | type-name   | annotation |
+      | entity    | person      | card(1, 2) |
+      | relation  | description | card(1, 2) |
+
+  Scenario Outline: <root-type> types can override inherited pure plays as plays with @<annotation>s
+    When put relation type: contract
+    When relation(contract) create role: contractor
+    When relation(contract) set annotation: @abstract
+    When put relation type: marriage
+    When relation(marriage) create role: spouse
+    When relation(marriage) set supertype: contract
+    When <root-type>(<supertype-name>) set annotation: @abstract
+    When <root-type>(<supertype-name>) set plays: contract:contractor
+    When <root-type>(<subtype-name>) set plays: marriage:spouse
+    When <root-type>(<subtype-name>) get plays(marriage:spouse) set override: contract:contractor
+    When <root-type>(<subtype-name>) get plays(marriage:spouse) set annotation: @<annotation>
+    Then <root-type>(<subtype-name>) get plays overridden(marriage:spouse) get label: contract:contractor
+    Then <root-type>(<subtype-name>) get plays(marriage:spouse) get annotations contain: @<annotation>
+    Then <root-type>(<subtype-name>) get plays(contract:contractor) get annotations do not contain: @<annotation>
+    Then <root-type>(<subtype-name>) get plays contain:
+      | marriage:spouse |
+    Then <root-type>(<subtype-name>) get plays do not contain:
+      | contract:contractor |
+    When transaction commits
+    When connection open read transaction for database: typedb
+    Then <root-type>(<subtype-name>) get plays overridden(marriage:spouse) get label: contract:contractor
+    Then <root-type>(<subtype-name>) get plays(marriage:spouse) get annotations contain: @<annotation>
+    Then <root-type>(<subtype-name>) get plays(contract:contractor) get annotations do not contain: @<annotation>
+    Then <root-type>(<subtype-name>) get plays contain:
+      | marriage:spouse |
+    Then <root-type>(<subtype-name>) get plays do not contain:
+      | contract:contractor |
+    Examples:
+      | root-type | supertype-name | subtype-name | annotation |
+      | entity    | person         | customer     | card(1, 2) |
+      | relation  | description    | registration | card(1, 2) |
+
+    # TODO: Maybe it should be rejected?
```

Comment:
?

Reply:
Once we pick a policy on behaviour we can decide

Reply:
This case is more about just ability to override, I rewritten this test to check that annotation is persistent after all these operations. It uses `set`, and we agreed that `set` can be used multiple times with the same target, so I'll remove the TODO.